src/main/java/fruitshop/model/FruitTransaction.java

```diff
@@ -0,0 +1,69 @@
+package fruitshop.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the start of the constructor.